packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -49,8 +49,11 @@ class GestureArenaEntry {
   ///
   /// It's fine to attempt to resolve a gesture recognizer for an arena that is
   /// already resolved.
-  void resolve(GestureDisposition disposition) {
-    _arena._resolve(_pointer, _member, disposition);
+  ///
+  /// If a [bid] is provided, the highest bid over 1.0 will be awarded victory upon
```

Comment:
There should be some documentation around how would one calculate the bid, According to the previous comments, bid = current value / goal. Can you add some documentation to explain that?

Reply:
also what does it mean for the gesture that is not determine by distance.

Reply:
also there should be a breadcumb to which gestures use the bid.